pkg/agent/cache/cache.go

```diff
@@ -61,6 +63,22 @@ func (c *cacheImpl) Entry(selectors []*common.Selector) (entry []CacheEntry) {
 	return nil
 }
 
+// PowerEntries takes a slice of selectors, and works through all the combinations in order to
+// find matching cache entries
+func (c *cacheImpl) PowerEntries(selectors []*common.Selector) (entries []CacheEntry) {
```

Comment:
Oops... should we update this one too? `MatchingEntries`? Sorry about that

Reply:
No worries, I missed it too.